homeassistant/util/dt.py

```diff
@@ -41,6 +41,18 @@ def utcnow():
     return dt.datetime.now(UTC)
 
 
+def beatsnow():
```

Comment:
Whatever. I'll keep the function in place in my fork as I don't have to worry much about memory usage on my system. At least care to reuse the proper code for calculating .beats so it does not display 1000+ .beats after 11pm UTC.